### PR TITLE
Altering theme for more subtle alerts / labels / buttons

### PR DESCRIPTION
**Pull request by mistercrunch**
@ascott @williaster The alerts were rough on my eyes and desire a more sober theme (at least for now). 

I threw in a `/caravel/theme/` endpoint to make it easier to iterate on css/less/themes.

screen shot 2016-07-21 at 9 32 27 am
screen shot 2016-07-21 at 9 32 20 am
screen shot 2016-07-21 at 9 32 15 am

### CODE REVIEW

File: caravel/templates/caravel/theme.html [hunk 1]
```diff
@@ -0,0 +1,605 @@
...
...u" value="false">         <param name="quality" value="best">         <param name="bgcolor" value="#ffffff">         <param name="wmode" value="transparent">         <param name="flashvars" value="trustedOrigins=getbootstrap.com%2C%2F%2Fgetbootstrap.com%2Chttps%3A%2F%2Fgetbootstrap.com">         <embed src="/assets/flash/ZeroClipboard.swf?noCache=1469113730367" loop="false" menu="false" quality="best" bgcolor="#ffffff" width="100%" height="100%" name="global-zeroclipboard-flash-bridge" allowscriptaccess="sameDomain" allowfullscreen="false" type="application/x-shockwave-flash" wmode="transparent" pluginspage="http://www.macromedia.com/go/getflashplayer" flashvars="trustedOrigins=getbootstrap.com%2C%2F%2Fgetbootstrap.com%2Chttps%3A%2F%2Fgetbootstrap.com" scale="exactfit">                </object></div><svg xmlns="http://www.w3.org/2000/svg" width="1140" height="500" viewBox="0 0 1140 500" preserveAspectRatio="none" style="display: none; visibility: hidden; position: absolute; top: -100%; left: -100%;"><defs><style type="text/css"></style></defs><text x="0" y="57" style="font-weight:bold;font-size:57pt;font-family:Arial, Helvetica, Open Sans, sans-serif">Thirdslide</text></svg></body>
```
Comment (ascott) [inline caravel/templates/caravel/theme.html]: what does this line do?

Comment (williaster) [inline caravel/templates/caravel/theme.html]: was also curious about this

**Comment by ascott** [comment]
i like the idea of a theme template to test our css changes, and maybe also add other components to in the future, but i think it would be nice to remove some of the bootstrap specific text. also looks like it's throwing some errors we should fix.

could we also remove all the links to bootstrap docs?

screenshot 2016-07-21 11 14 40

maybe we could just include the elements we use in caravel, like menus, buttons, alerts, tables? _simplify_ :tada:

**Comment by ascott** [comment]
do we want the query button to use the primary color here? i think the light grey looks better and doesn't draw too much attention.
 
before:
screenshot 2016-07-21 12 56 17

after:
screenshot 2016-07-21 12 55 47

looks like we need to treat links in alerts differently in alerts with these new lighter colors: 
screenshot 2016-07-21 12 53 21

**Comment by williaster** [comment]
I like the idea of the theme test page as well but agree it could be nice to simplify and include only the components we utilize in Caravel. alanna as we move to React do you think it would make sense to use storybook for this type of thing as well?

I like the muted / grey query button as well. I think once there is more front end logic, having it "pop" from grey to another color on a query builder change would be solid / help the user know a parameter changed. 

nice catch on the link text within alerts!

File: caravel/templates/caravel/theme.html (hunk 1 above)
Comment (mistercrunch) [inline caravel/templates/caravel/theme.html]: not sure, it came from wget-ing the bootstrap theme demo page... I'm taking it out

**Comment by mistercrunch** [comment]
* changed button-primary to more sober grey instead of brand-primary
* remove carousel from theme demo page and other useless items

I can't figure out how to get rid of the 404s and how the white link is alterable... Will need help to address those.

**Comment by williaster** [comment]
I was searching a bit for the link color but no 🎲 yet, I think it must be one of the `@...-link-color-...` variables.

**Comment by mistercrunch** [comment]
Alright I cleaned things up (js on main page) and forced a hack for the links to not render in white in Alerts.

**Pull request merged**